### PR TITLE
Add: Home page

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

**Review by yash9263 — APPROVED** [review]

**Comment by yash9263** [comment]
Everything looks good to me nice clean ui

File: style.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+/* footer  */
+.home_footer {
+  justify-content: space-around;
+  align-items: center;
+  color: var(--text-primary);
+}
+.footer_ul > li {
+  padding: 0.5rem 0;
```
Comment (yaswanthmyneni) [inline style.css:93]: Changes: use px units for less than 16px.

File: styles/nav.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  border-radius: 0;
+}
+.login_banner > img {
+  filter: brightness(50%) grayscale(100%);
+}
+.login_banner > .nav_brand_text {
+  position: absolute;
+  color: var(--bg-color);
+  overflow: hidden;
+  z-index: 2;
```
Comment (yaswanthmyneni) [inline styles/nav.css]: Changes: use css variables for z-index

File: index.html [hunk 3]
```diff
@@ -0,0 +1,444 @@
+<!DOCTYPE html>
```
Comment (yaswanthmyneni) [inline index.html:1]: UX improvement: rename the "buy now" button in the card to "add to wishlist". And make "add to cart" as a primary button.
Screenshot 2022-02-20 at 1 06 40 AM

**Review by yaswanthmyneni — CHANGES_REQUESTED** [review]
UI looks good. After changing minor issues, you can merge pr.

File: index.html (hunk 3 above)
Comment (gauravsinhaweb) [reply index.html:1]: The button 'buy now' and 'add to cart' will redirect to the same path. That is intentional, thanks for the suggestion.

File: styles/nav.css (hunk 2 above)
Comment (gauravsinhaweb) [reply styles/nav.css]: @yaswanthmyneni, Done, can you approve the changes.

File: style.css (hunk 1 above)
Comment (gauravsinhaweb) [reply style.css:93]: I want to keep padding relative to the mobile device, so used rem here.

**Review by yaswanthmyneni — APPROVED** [review]

**Pull request merged**